backend/api/auth.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express");
+const router = express.Router();
+const request = require('request');
+const path = require('path');
+require('dotenv').config();
```

Comment:
This line is a duplicate of [server.js:5](https://github.com/COP4331-Group3-Fall2021/U-Ride/blob/main/backend/server.js#L5)